core/api/src/domain/bitcoin/lnurl/index.types.d.ts

```diff
@@ -0,0 +1,6 @@
+interface ILnurlPayService {
+  fetchInvoiceFromLnAddressOrLnurl(args: {
+    amount: BtcPaymentAmount
+    lnAddressOrLnurl: string
+  }): Promise<string | ApplicationError>
```

Comment:
application error is not part of domain, please use the base service error like in other implementations